src/fitsviewer.py

```diff
@@ -504,9 +510,33 @@ def __init__(self, parent=None):
         self.active_plot_window = None
 
     def on_indices_toggled(self, state):
-
         self.ui.arrayIndices.setEnabled(state)
 
+    def create_analysis_template(self):
+        index = self.ui.sections.currentIndex()
+        #data_str=str(self.ui.sections.model().index(index.row(),index.column()).data())
+        template='''import numpy\nfrom matplotlib import pyplot as plt\nfrom astropy.io import fits\nhdul = fits.open("{fname}")\nprint(hdul.info())\n'''
```

Comment:
Can you put this text into a separate file or, if not easily possible, at the top as a multi-line string literal with proper linebreaks?

Reply:
It creates a script template and copies it to the clipboard.  Then one could paste it to  for example jupyter notebook. 
This feature can be removed if you want.  Actually,  I wanted to integrate a python terminal into the GUI.